demo/minidemo.html

```diff
@@ -4,14 +4,14 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <script src='./lib/openseadragon/openseadragon.js' ></script>
-  <script src='./lib/loading/loading.js'></script>
-  <script src='./js/util.js' type='text/javascript'></script>
-  <script src='./core/Store.js'></script>
-  <script src='./core/CaMic.js'></script>
-  <script src='./lib/openseadragon-imaginghelper.min.js'></script>
+  <script src='../core/openseadragon/openseadragon.js' ></script>
+  <script src='../components/loading/loading.js'></script>
+  <script src='../common/util.js' type='text/javascript'></script>
```

Comment:
```suggestion
  <script src='../common/util.js' type='text/javascript'></script>
  <script type='text/javascript' src='../common/ajv.js'></script>
  <script type='text/javascript' src='../core/Validation.js'></script>
```

Reply:
Also, I don't understand the need for the demo. as I was going through I find a few errors so fixed it.